water/main.py

```diff
@@ -257,11 +285,19 @@ def animate():
 demand_weight_slider.on_change('value', update_node_sizes)
 demand_weight_slider.value = node_demand_weighting
 
+dropdown = Dropdown(label="Pollution Injection Location", button_type="primary", menu=menu)
+dropdown.on_change('value', update_colors)
```

Comment:
I think this an extra unused dropdown?

Reply:
Removed. I think they must have been left over from when I rebased this branch.